### PR TITLE
Enhancement/focal support

### PR DESCRIPTION
**Pull request by AGS4NO**
Update CI to support focal

Belongs to https://github.com/dnsimple/dnsimple-ops/issues/2550

## :shipit: Deploy Steps:

- [x] Merge and pull latest
- [ ] [Community cookbook release](https://dnsimple.atlassian.net/wiki/spaces/DEV/pages/347242670/Cookbook+Workflow+-+Community#Deploying-Approved-Changes)

## :shipit: Verification Steps

- [ ] Ensure tests pass

### CODE REVIEW

**Review by san983 — APPROVED** [review]

File: metadata.rb [hunk 1]
```diff
@@ -7,9 +7,9 @@
 source_url       'https://github.com/dnsimple/chef-dnsimple'
 version          '3.3.0'
 
-chef_version '>= 13.9'
+chef_version '>= 16'
```
Comment (onlyhavecans) [inline metadata.rb:10]: I think we should keep 15 support for now

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ If you used the dnsimple_record resource, you'll want to do the following to mig
 If you used the dnsimple_certificate resource, you'll want to do the following to migrate:
 
 * Rename `certificate_common_name` properties to `common_name`
-* Update your `expires_on` to be a parsable date string (see examples below)
+* Update your `expires_at` to be a parsable timestamp (see examples below)
```
Comment (onlyhavecans) [inline README.md:17]: The Resource Properties section needs updating as well

**Review by onlyhavecans — CHANGES_REQUESTED** [review]
Do not mark files All Rights Reserved in this Apache Licenced Cookbook, that is only for internal cookbooks

This also is an API-breaking change with certificates. Can we break that change out into a separate PR so we can make it clear where this can break people's use of it?

File: metadata.rb (hunk 1 above)
Comment (AGS4NO) [reply metadata.rb]: Set constraint to '>= 15'

**Comment by AGS4NO** [comment]
@onlyhavecans I have broken the changes out for a separate release and locked the dnsimple gem version to be compatible with the current codebase.

**Review by onlyhavecans — APPROVED** [review]

**Pull request merged**